src/core/sys/posix/signal.d

```diff
@@ -1687,6 +1711,8 @@ else version( CRuntime_UClibc )
                             // this signal, as defined in <errno.h>
         int si_code;        // Signal code
 
+        version ( MIPS32 ) int[__SI_MAX_SIZE / int.sizeof - __SI_PAD_SIZE - 3] __pad0;
```

Comment:
I'd prefer not to have version conditions nested in as per previous comment.

Reply:
Didn't wanted to duplicate that huge structure...